typescript/cli/src/deploy/agent.ts

```diff
@@ -65,7 +65,7 @@ export async function runKurtosisAgentDeploy({
   const kurtosisCloudUrl = getKurtosisCloudUrl(base64EncodedPackageConfig);
 
   const kurtosisCloudLink = terminalLink(
-    'Cmd+Click or Ctrl+Click here',
+    'Kurtosis Cloud Link 🔗 (cmd+click or ctrl+click here)',
```

Comment:
could use the unicode symbols
Command (or Cmd) ⌘
Control (or Ctrl) ⌃

Reply:
Personally I think `⌘+click or ⌃+click`, `⌘click or ⌃click`, etc. are less readable than `cmd+click or ctrl+click` since we are asking for a "click" and not another key or character input